src/sidebar/helpers/test/mentions-test.js

```diff
@@ -556,4 +556,14 @@ describe('toPlainTextMention', () => {
       assert.equal(toPlainTextMention(user, mentionMode), expectedMention);
     });
   });
+
+  it('removes square brackets in display-name mode', () => {
+    const user = {
+      userid: 'acct:jane_doe@foo.com',
+      username: 'jane_doe',
+      displayName: 'Jane [Doe]',
```

Comment:
Maybe worth adding a second bracket-ed term to verify that all brackets are removed, not just the first occurrence.